cmd/changefeed.toml

```diff
@@ -32,9 +32,9 @@ dispatchers = [
 	{matcher = ['test3.*', 'test4.*'], dispatcher = "rowid"},
 ]
 # 对于 MQ 类的 Sink，可以指定消息的协议格式
-# 协议目前支持 default, canal, avro 和 maxwell 四种，default 为 ticdc-open-protocol
+# 协议目前支持 default, canal 两种，default 为 ticdc-open-protocol
```

Comment:
Why disable avro and maxwell?

Reply:
This change seems to be introduced unintentionally. Let me review the code from the beginning.

Reply:
Reverted.